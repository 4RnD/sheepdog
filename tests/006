#!/bin/bash

# Test launching sheeps simultaneously

seq=`basename $0`
echo "QA output created by $seq"

here=`pwd`
tmp=/tmp/$$
status=1        # failure is the default!

# get standard environment, filters and checks
. ./common.rc
. ./common.filter

_cleanup

_start_sheep 0
_wait_for_sheep 1

# start Sheepdog with one node
$COLLIE cluster format
sleep 1

# launch sheeps simultaneously
for i in `seq 1 9`; do
    _start_sheep $i
done

_wait_for_sheep 10

echo check whether all nodes have the same cluster info
for i in `seq 0 9`; do
    $COLLIE cluster info -p 700$i | _filter_cluster_info > $STORE/cinfo.$i
done
for i in `seq 1 9`; do
    diff -u $STORE/cinfo.0 $STORE/cinfo.$i
done
