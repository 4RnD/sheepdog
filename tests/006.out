QA output created by 006
using backend farm store
check whether all nodes have the same cluster info
